### ISSUE TITLE
Name this group!

### ISSUE THREAD

**Issue by saulshanabrook** [saulshanabrook/jupyterlab-accessibility#21]
We should come up with a fun name for this group, for our github organization, website, etc

**Comment by adelaide101** [comment]
hacklife 
h@ck3r$
approachable_hacking
jupyterlabs_fanclub

....wait, did we already pick the name jupyterlabs_accessibility? if so, i like that name

**Comment by saulshanabrook** [comment]
@adelaide101 yeah for now it's jupyterlab-accessability! We can keep that and even make an "organization" for it, so it won't be under my name anymore

**Comment by saracoderbrown** [comment]
local coders
